util/util.go

```diff
@@ -5,13 +5,44 @@ import (
 	"fmt"
 
 	ipfslite "github.com/hsanjuan/ipfs-lite"
+	"github.com/ipfs/go-datastore"
 	"github.com/ipfs/go-log"
 	crypto "github.com/libp2p/go-libp2p-core/crypto"
+	"github.com/libp2p/go-libp2p-core/host"
+	dht "github.com/libp2p/go-libp2p-kad-dht"
 	"github.com/multiformats/go-multiaddr"
+	"github.com/textileio/grpc-ipfs-lite/server"
 )
 
-// NewPeer creates a new ipfslite.Peer
-func NewPeer(ctx context.Context, datastorePath string, port int, debug bool) (*ipfslite.Peer, error) {
+type manager struct {
```

Comment:
I'm creating this struct type here and saying it will conform to server.PeerManager